README.md

```diff
@@ -71,7 +71,8 @@ Specify what crate to add:
                             `cargo add bitflags@0.3.2`.
     --git <uri>             Specify a git repository to download the crate from.
     --path <uri>            Specify the path the crate should be loaded from.
-
```

Comment:
I think we should follow cargo in describing the feature flag:
```bash
$ cargo build --help 2>&1 | grep -e "--features"
    --features FEATURES          Space-separated list of features to also build
```
(and maybe replace `also build` with `add`)